app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -1,4 +1,4 @@
-module TwoFactorAuthenticatableMethods # rubocop:disable Metrics/ModuleLength
+module TwoFactorAuthenticatableMethods
```

Comment:
we could probably collapse this back into `TwoFactorAuthenticatable` because literally that module just includes this one, right?

Reply:
That is true, though I don't think satisfying this linter was the role of this module. I'll leave that to a follow-on PR.